atd-vze/src/views/Fatalities/fatalityGridTableParameters.js

```diff
@@ -68,3 +71,202 @@ export const fatalityGridTableColumns = {
     type: "String",
   },
 };
+
+export const fatalityGridTableAdvancedFilters = {
+  groupUnits: {
+    icon: "bicycle",
+    label: "Unit Type",
+    filters: [
+      {
+        id: "motor_vehicle",
+        label: "Motor Vehicle Fatality",
+        filter: {
+          where: [
+            {
+              "_or: [ { _and: [ { person: { unit: { unit_desc_id: { _eq: 1 } } } }, { _and: [ { person: { unit: { veh_body_styl_id: { _neq: 71 } } } }, { person: { unit: { veh_body_styl_id: { _neq: 90 } } } } ] } ] }, { _and: [ { primaryperson: { unit: { unit_desc_id: { _eq: 1 } } } }, { _and: [ { primaryperson: { unit: { veh_body_styl_id: { _neq: 71 } } } }, { primaryperson: { unit: { veh_body_styl_id: { _neq: 90 } } } } ] } ] } ]": null,
```

Comment:
shoutout to @tillyw for some of these `where` filters from this [PR](https://github.com/cityofaustin/atd-vz-data/pull/1226) that I was able to adapt for the Fatalities page. the unit type filters look prettyyy crazy because I had to use an `or` operator since the record could be a person or a primary person.